app/components/UI/Stake/Views/UnstakeInputView/UnstakeInputView.tsx

```diff
@@ -75,14 +84,32 @@ const UnstakeInputView = () => {
     );
   }, [navigation, theme.colors, title]);
 
-  const handleUnstakePress = useCallback(() => {
-    navigation.navigate('StakeScreens', {
-      screen: Routes.STAKING.UNSTAKE_CONFIRMATION,
-      params: {
-        amountWei: amountWei.toString(),
-        amountFiat: fiatAmount,
-      },
-    });
+  const handleUnstakePress = useCallback(async () => {
+    const isStakingDepositRedesignedEnabled =
+      confirmationRedesignFlags?.staking_transactions;
+
+    if (isStakingDepositRedesignedEnabled) {
+      await attemptUnstakeTransaction(
+        amountWei.toString(),
+        activeAccount?.address as string,
+      );
+
+      navigation.navigate('StakeScreens', {
+        screen: Routes.STANDALONE_CONFIRMATIONS.STAKE_WITHDRAWAL,
```

Comment:
Let's keep the naming consistent for staking actions. We stake, unstake, and claim.

```suggestion
        screen: Routes.STANDALONE_CONFIRMATIONS.STAKE_UNSTAKE,
```

Reply:
We're keeping the naming consistent with the STANDALONE_CONFIRMATIONS object in which there already is a STAKE_DEPOSIT entry, and the corresponding component is also called StakingDeposit.